app/templates/components/mandatenbeheer/mandataris-duplicate-selector.hbs

```diff
@@ -5,6 +5,7 @@
         @options={{this.options}}
         @selected={{this._duplicatedMandataris}}
         @onChange={{action "select"}}
+        {{!template-lint-disable no-action}}
```

Comment:
Temporary since this component isn't actually used by the app as far as I can tell.